charts/plugin-br-pix-switch/templates/plugin-br-pix-switch/hpa.yaml

```
@@ -0,0 +1,32 @@
+  name: {{ include "plugin-br-pix-switch.fullname" . }}
+  labels:
+    {{- include "plugin-br-pix-switch.labels" (dict "context" . "name" .Values.pixSwitch.name ) | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "plugin-br-pix-switch.fullname" . }}
+  minReplicas: {{ .Values.pixSwitch.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.pixSwitch.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.pixSwitch.autoscaling.targetCPUUtilizationPercentage }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)